src/gax-internal/grpc-server/build.rs

```diff
@@ -15,7 +15,7 @@
 fn main() {
     #[cfg(feature = "_generate-protos")]
     {
-        tonic_build::configure()
+        tonic_prost_build::configure()
```

Comment:
Presumably we need similar changes in `sidekick` so it can call the right version of `Tonic` and `Prost`?

Reply:
Nope, the generated code (proto messages) only uses `prost_build`, which is unaffected. Only the `grpc-server` (a gRPC server, duh) actually needs `tonic_prost_build`.